.travis.yml

```diff
@@ -1,7 +1,10 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
-language: objective-c
+language: generic
```

Comment:
so I take it we no longer need to pretend we're using objective-c to get an OS-X build machine?